src/components/CCIP/Landing/ccip-landing.astro

```diff
@@ -29,14 +29,16 @@ const supportedTokens = getAllSupportedTokens({
 
 const tokens = Object.keys(supportedTokens).sort((a, b) => a.localeCompare(b, undefined, { sensitivity: "base" }))
```

Comment:
we were already doing sort here. Why do we have to do below too?! I would keep just once

Reply:
another thought.. Should we not actually sort it at the initial methods (so within the "getTokens" and "getNetworks" instead than do it in this individual files?

Reply:
Yes the `.sort()` is not needed here
about your second comment: because `getAllSupportedTokens` is a function that returns an object and it's not guaranteed to organize an object so I'm doing it here in the top level of the page
Also it's being used in other places all over the app not just CCIP so it will be hard to change this function